src/components/Modals/Onboarding.vue

```diff
@@ -10,197 +14,56 @@
-                <div class="onboarding-add" @click="addMajor">
-                  Add
-                </div>
-                <div class="onboarding-remove" @click="removeMajor" :class="{ 'onboarding--hidden': displayOptions.major.length <= 1 }">
-                  Remove
-                </div>
-              </div>
-              -->
-            </div>
-            <!--
-            <div class="onboarding-inputWrapper onboarding-inputWrapper--college">
-              <label class="onboarding-label">Your Minor (optional)</label>
-              <div class="onboarding-selectWrapper">
-                <div
-                  class="onboarding-select onboarding-input"
-                  id="minor"
-                  v-for="(options, index) in displayOptions.minor"
-                  :key = index
-                  :style="{ borderColor: options.boxBorder }"
-                  v-click-outside:[index]="closeMinorDropdownIfOpen"
-                >
-                  <div class="onboarding-dropdown-placeholder minor-wrapper" @click="showHideMinorContent(index)">
-                    <div
-                      class="onboarding-dropdown-placeholder minor-placeholder"
-                      id="minor-placeholder"
-                      :style="{ color: options.placeholderColor }"
-                    >
-                      {{ options.placeholder }}
-                    </div>
-                    <div
-                      class="onboarding-dropdown-placeholder minor-arrow"
-                      id="minor-arrow"
-                      :style="{ borderTopColor: options.arrowColor }"
-                    ></div>
-                  </div>
-                  <div
-                    class="onboarding-dropdown-content minor-content"
-                    id="minor-content"
-                    v-if="options.shown"
-                  >
-                    <div
-                      v-for="(minor, acronym) in minors"
-                      :key="acronym"
-                      :id="minor"
-                      class="onboarding-dropdown-content-item"
-                      @click="selectMinor(minor, acronym, index)"
-                    >
-                      {{ minor }}
-                    </div>
-                  </div>
-                </div>
-              </div>
-              <div class="onboarding-addRemoveWrapper" :class="{ 'onboarding--hidden': Object.keys(minors).length <= 0}">
-                <div class="onboarding-add" @click="addMinor">
-                  Add
-                </div>
-                <div class="onboarding-remove" @click="removeMinor" :class="{ 'onboarding--hidden': displayOptions.minor.length <= 1 }">
-                  Remove
-                </div>
-              </div>
-            </div>
-            -->
+        <div v-else class="onboarding-bottom--section">
+          <div class="onboarding-bottom--contents" @click="$emit('cancelOnboarding')">
```

Comment:
A bit of a nitpick, but might be valuable to put this $emit in its own function since its called in a couple different places.